modyn/supervisor/supervisor.py

```diff
@@ -222,6 +243,13 @@ def get_dataset_selector_batch_size(self) -> None:
     def validate_system(self) -> bool:
         return self.dataset_available() and self.grpc.trainer_server_available()
 
+    def _run_tensorboard(self, port: int) -> None:
+        logger.info(f"Starting up tensorboard on port {port}.")
+        try:
+            subprocess.call(f"tensorboard --logdir {self.eval_directory} --bind_all --port={port}".split(" "))
```

Comment:
Can we avoid subprocess? Rather, I would prefer to run tensorboard within Python, if possible. I did a quick search and found that: https://stackoverflow.com/questions/42158694/how-to-run-tensorboard-from-python-scipt-in-virtualenv

Reply:
I agree. Had some issues with excessive logging of tensorboard. This should now work :)